controle_atividades/settings.py

```diff
@@ -131,3 +131,6 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+STATICFILES_DIRS = [
```

Comment:
Quando vc adiciona o `Yarn` ele passa a controlar os arquivos statics do sistema, então não precisa mais baixar arquivo por aquivo e colocar na pasta `static` basta instalar pelo yarn e depois importar no HTML normalmente.

Reply:
o yarn funciona tipo um cdn?

Reply:
CDN: você pega links da nuvem e coloca no projeto
YARN: você baixa as libs que for usar porem tais libs não vão para o servidor final

Bom de usar yarn é que tu consegue programar sem internet, basta baixar as libs que se projeto precisa e pronto, coisa que CDN não rola.

APOS INSTALAR O YARN

como ta configurado já, tu só precisa abrir o terminal na pasta do projeto e rodar `yarn` que ele faz toda a brincadeira pra vc.